classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_students(student)
+    @students.push(student)
+    student.Classroom = self
```

Comment:
- [ ] To avoid unnecessary syntax errors, kindly make this `classroom`, with lower case.